CONTRIBUTING.md

```diff
@@ -39,8 +40,70 @@ git merge upstream/master
 # push the changes to your master branch to the remote repository
 git push
 ```
+
 You do not want to commit changes to your `master` branch, any changes to your `master` branch should come from the upstream `master` branch.
 
+### Size and composition of pull requests
+
+An ideal pull request is small enough for reviewers to review the code in detail and focused on a single area or, if adding a new file entirely, a single file.
+Implementing an analysis will often require more than one notebook or script to be added to the repository.
+In these cases, it is desirable to submit _multiple pull requests_ for an analysis rather than a single, large pull request when an analysis is completed.
```

Comment:
```suggestion
It is best to submit _multiple pull requests_ for these analyses rather than a single, large pull request when an analysis is completed.
```